windows-csharp/ShoutLib/ShoutLib.cs

```diff
@@ -90,80 +91,83 @@ public Shouter(LogWriter logWriter = null)
         }
 
         /// <summary>
-        /// Waits for a task on the queue.  Converts the text to uppercase and posts the results
-        /// to the browser's topic.
+        /// Waits for a shout request to arrive in the Pub/Sub subscription.
+        /// Converts the text to uppercase and posts the results 
+        /// back to the website.
         /// </summary>
         /// <returns>
-        /// The number of tasks pulled from the queue, or -1 if an expected error
-        /// occurred.
+        /// The number of shout requests pulled from the subscription, or -1

```

Comment:
As in my comment above, maybe cast in terms of pubsub *messages*.  e.g. "..number of 'shout request' messages.."?  
Otherwise on a quick read they may be thinking these are 'requests' to grab from the topic, or something along those lines.